Assets/MRTK/Services/CameraSystem/MixedRealityCameraSystem.cs

```diff
@@ -78,6 +78,54 @@ public bool IsOpaque
             }
         }
 
+        /// <inheritdoc />
+        public bool ReadingModeEnabled
+        {
+            get
+            {
+                ProjectionOverride projectionOverride;
+                if (TryGetProjectionOverrideComponent(out projectionOverride, false))
+                {
+                    return projectionOverride.ReadingModeEnabled;
+                }
+                return false;
+            }
+            set
+            {
+                ProjectionOverride projectionOverride;
+                if (TryGetProjectionOverrideComponent(out projectionOverride, true))
+                {
+                    projectionOverride.ReadingModeEnabled = value;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Helper to get the <see cref="ProjectionOverride"> component that's attached to the main camera
+        /// </summary>
+        /// <param name="projectionOverride">The <see cref="ProjectionOverride"> component if there is one</param>
+        /// <param name="createIfAbsent">Create the <see cref="ProjectionOverride"> if it's not there</param>
+        /// <returns>
+        /// false if there was no ProjectionOverride component and we didn't create one
+        /// true if the out param projectionOverride is set to the ProjectionOverride attached to the main camera
+        /// </returns>
+        private bool TryGetProjectionOverrideComponent(out ProjectionOverride projectionOverride, bool createIfAbsent)
+        {
+            projectionOverride = CameraCache.Main.GetComponent<ProjectionOverride>();
```

Comment:
Since most of the interaction is on this MonoBehaviour component, can we instead treat this as a Windows-specific camera setting, like we do for the reprojection component?

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/7c9fdf9a56291700dbcfd9916805c4329045d534/Assets/MRTK/Providers/WindowsMixedReality/Shared/BaseWindowsMixedRealityCameraSettings.cs#L83-L90

Reply:
`ReadingModeEnabled` can be in the profile, then all other interactions can take place directly with `ProjectionOverride`